x-pack/libbeat/management/blacklist.go

```diff
@@ -98,6 +98,11 @@ func (c *ConfigBlacklist) isBlacklisted(blockType string, block *api.ConfigBlock
 			if c.isBlacklistedBlock(pattern, segments, cfg.Config) {
 				return true
 			}
+		} else if strings.HasPrefix(field, ".") {
+			segments := strings.Split(field[1:], ".")
+			if c.isBlacklistedBlock(pattern, segments, cfg.Config) {
+				return true
+			}
 		}
 	}
 
```

Comment:
I know we have a more integrated test with the manager but could we add a simple unit test in blacklist_test.go, In case we want to reuse this object outside of CM.